README.md

```diff
@@ -1,5 +1,8 @@
 # Hibernate Second Level Cache
 
+<a href="https://github.com/actions/toolkit"><img alt="GitHub Actions status" src="https://github.com/hazelcast/hazelcast-hibernate5/workflows/build/badge.svg"></a>
```

Comment:
IMHO `https://github.com/hazelcast/hazelcast-hibernate5/actions?query=event%3Apush+branch%3Amaster` would be more relevant link here.

Reply:
Official examples point always at toolkit, but it's a good idea actually.